airflow/config_templates/default_webserver_config.py

```diff
@@ -130,3 +130,16 @@
 # APP_THEME = "superhero.css"
 # APP_THEME = "united.css"
 # APP_THEME = "yeti.css"
+
+# ----------------------------------------------------
+# Simple auth manager config
+# ----------------------------------------------------
+# This list contains the list of users and their associated role in simple auth manager.
+# If the simple auth manager is used in your environment, this list controls who can access the environment.
+# Example:
+# [{
+#     "username": "admin",
+#     "password": "admin",
+#     "role": "admin",
```

Comment:
As https://github.com/apache/airflow/pull/42004/files#diff-c8acbf9922a035c99d5347625cc8c008a0a2a2c1e1e451699ee8145b7fa85265R63 is upper-case this is only working if upper-case as well.
```suggestion
#     "role": "ADMIN",
```

Reply:
I fixed it by adding a `upper()` in the code :)